app/javascript/dashboard/routes/dashboard/helpcenter/components/PortalSwitch.vue

```diff
@@ -111,12 +127,14 @@ export default {
       });
       this.$emit('open-portal-page');
     },
-    isLocaleActive(code) {
-      const isPortalActive = this.portalSlug === this.portal.slug;
+    isLocaleActive(code, slug) {
+      const isPortalActive = this.portal.slug === slug;
       const isLocaleActive = this.portal?.meta?.default_locale === code;
```

Comment:
```suggestion
      const isLocaleActive = this.$route.params.locale === code;
```

Instead of taking the current active locale from the store can we take it from route params? So I think the portal switch active state will work properly. Currently, there is some issue.

![image](https://user-images.githubusercontent.com/64252451/188270793-065d166d-8885-4fc0-bf33-3188200ce628.png)

Reply:
Yes, We will do this in a diffrerent PR. We need to do it like `selectedPortal` since there are URL's like `app.chatwoot.com/accounts/1/portals` where param is not available.